tests/test_client.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import rdflib
+import requests
+
+from fdpclient.client import Client
+
+# TODO
+# - add tests on fdp
+
+# NOTE: change IP address to domain name after solving FDP server config issue
+base_url = 'http://145.100.57.30'
+catalogID = 'catalog01'
+data_url = base_url + '/catalog/' + catalogID
+
+@pytest.fixture(scope='class', autouse=True)
```

Comment:
Are you sure you don't want to clean up between tests?

Reply:
The tests within one class are not independent, e.g. the `read` needs the data created by `create`.  It's also fine to make them independent.

Reply:
Ah. You may want to reconsider that. I can tell you from experience that if something goes wrong it is very annoying to debug...